### ISSUE TITLE
Issues with eye of the storm

### ISSUE THREAD

**Issue by username-is-not-taken** [SunwellWoW/Sunwell-TBC-Bugtracker#3844]
Description: 
You can't cap bases when mounted.
There is no flag icon on the map when someone is carrying the flag.
I can't remember if there should be one or not, but there's no flag icon next to the resources when someone is carrying the flag (like in WSG).

Source (you should point out proofs of your report, please give us some source): 

![eots1](https://user-images.githubusercontent.com/45629295/50456415-9ef0e100-0954-11e9-8386-52b2c5c984eb.png)

![eots2](https://user-images.githubusercontent.com/45629295/50456419-a3b59500-0954-11e9-9cef-3c406e2ee4ca.png)

**Comment by Azmoddan** [comment]
About carrying the flag - there should be information on chat AND a flag icon on the map.
Capping the base should be also possible while mounted therefore I guess you might have encountered a visual bug maybe?

Need to fix.

**Comment by username-is-not-taken** [comment]
No, it's not a visual bug, I've tried to mount/dismount a couple of times when solo capping a base to see if the bar moves when mounted and it does not.
